content/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide.md

```diff
@@ -163,7 +163,7 @@ To share feedback about {% data variables.product.prodname_copilot_chat %}, you
 
 {% data variables.product.prodname_copilot_chat %} is available to all organizations and enterprises that have an active {% data variables.product.prodname_copilot_for_business %} license. You can enable or disable {% data variables.product.prodname_copilot_chat %} for your organization or enterprise in the {% data variables.product.prodname_copilot_for_business %} settings page.
 
-If {% data variables.product.prodname_copilot_chat_short %} is enbaled or disabled at the enterprise level, organizations within the enterprise cannot override the setting.
+If {% data variables.product.prodname_copilot_chat_short %} is enabled or disabled at the enterprise level, organizations within the enterprise cannot override the setting.
```

Comment:
#### Before
https://docs.github.com/en/enterprise-cloud@latest/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#enabling-or-disabling-github-copilot-chat
<img width="704" alt="image" src="https://github.com/github/docs/assets/2119212/e5e5838c-7d3f-4bba-94df-1a076fe287bd">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/enterprise-cloud@latest/copilot/github-copilot-chat/using-github-copilot-chat-in-your-ide#enabling-or-disabling-github-copilot-chat
<img width="704" alt="image" src="https://github.com/github/docs/assets/2119212/f980c090-88f7-4199-a7bc-66ecc04320c6">